client/core/blockchain.go

```diff
@@ -941,7 +939,7 @@ func writePositionalMetadata(batch kcoindb.Batch, block *types.Block, state *sta
 }
 
 func IsHead(block *types.Block, currentBlock *types.Block) bool {
```

Comment:
don't think this name is correct, it just check the blocks are same level

Reply:
IsParent?